Automattic/wp-calypso
# A4A > Marketplace: Add Standard agency hosting testimonial

## Proposed Changes

* This PR adds a testimonial to the Standard agency hosting

## Why are these changes being made?

* Update the testimonial section on the Standard agency hosting for the new hosting page.

## Testing Instructions

* Open A4A live link
* Go to Marketplace - Hosting - Standard agency hosting
* Verify the newly added testimonial is displayed as shown below. Design: 3JELpER0gWwwI1BMrTyGE0-fi-6836_73480

<img width="1728" alt="Screenshot 2024-08-05 at 9 11 58 AM" src="https://github.com/user-attachments/assets/b5e9794c-e83b-4af4-a2fb-c615092a1866">

<img width="1407" alt="Screenshot 2024-08-05 at 3 20 51 PM" src="https://github.com/user-attachments/assets/c87cb3f1-40a3-4f84-8c48-c3a90827207b">


## Pre-merge Checklist
